libs/go.json

```diff
@@ -221,5 +221,5 @@
   {"id": "go.golarm", "imports": ["github.com/msempere/golarm"], "name": "golarm", "repo": "msempere/golarm", "tags": ["alarms"], "tech": ["diagnostics"], "status": "model-ready"},
   {"id":"go.logrus","imports":["github.com/sirupsen/logrus"],"name":"logrus","repo":"Sirupsen/Logrus","tags":["logging"],"tech":["diagnostics"],"status":"awaiting-model"},
   {"id":"go.mux","imports":["mux"],"name":"mux","repo":"gorilla/mux","tags":["routing"],"tech":["go-web"],"status":"awaiting-model"},
-  {"id":"go.viper","imports":["github.com/spf13/viper"],"name":"viper","repo":"spf13/viper","tags":["configuration"],"tech":["utils"],"status":"awaiting-model"},
+  {"id":"go.viper","imports":["github.com/spf13/viper"],"name":"viper","repo":"spf13/viper","tags":["configuration"],"tech":["utils"],"status":"awaiting-model"}
```

Comment:
we actually have a category 'configuration' but maybe this is better. not sure.

Reply:
it was a comma-only change. I like to have a trailing comma, but it makes JSON invalid. @yaronskaya does it make any difference for training whether json files are valid JSONs or trailing comma doesn't make any difference for you?